Dockerfile

```diff
@@ -77,11 +77,11 @@ RUN apt-get update -y -q && \
     apt-get install -y clang-3.9 lldb-3.9 && \
     apt-get install -y libc6-i386  libsuitesparse-dev && \
     export LLVM_CONFIG=/usr/lib/llvm-3.9/bin/llvm-config && \ 
-    git clone --recursive https://github.com/simonsfoundation/CaImAn.git && \
+    git clone --recursive https://github.com/ecobost/CaImAn.git && \
```

Comment:
this should probably be forked to `github.com/cajal` or `/atlab` and cloned from there.